scripts/grabURI.coffee

```diff
@@ -1,107 +1,28 @@
 # Description:
```

Comment:
If we're leaving this blank, we should just kill the lines. Also, copyright/license header is missing